ansible/lemmy.yml

```diff
@@ -27,7 +28,18 @@
         - 'docker-compose'
         - 'docker.io'
         - 'certbot'
+
+  - name: install certbot-nginx on ubuntu < 20
+    apt:
+      pkg:
         - 'python-certbot-nginx'
+    when: not (ansible_distribution == 'Ubuntu' and ansible_distribution_version|version_compare('20.04', '>=')
```

Comment:
Could you please change remove the `not` and use `<` for the comparison?

Btw I noticed this exact problem while migrating dev.lemmy.ml to a new Ubuntu 20.04  server, so this is in fact a necessary change.